client/blocks/app-banner/index.jsx

```diff
@@ -190,9 +237,35 @@ export class AppBanner extends Component {
 				</Card>
 			</div>
 		);
+	};
+
+	render() {
+		if ( ! this.props.shouldDisplayAppBanner || this.state.isDraftPostModalShown ) {
+			return null;
+		}
+
+		const displayJetpackAppBranding = config.isEnabled( 'jetpack/app-branding' );
+
+		return displayJetpackAppBranding
+			? this.getJetpackAppBanner( this.props )
+			: this.getWordpressAppBanner( this.props );
 	}
 }
 
+function BannerIcon( { icon } ) {
+	useEffect( () => {
+		lottie.loadAnimation( {
+			container: document.querySelector( '.app-banner__icon' ),
+			renderer: 'svg',
+			loop: false,
+			autoplay: true,
+			path: icon,
+		} );
+	}, [] );
```

Comment:
To avoid unexpected outcomes and address linter warnings, we should probably list all the dependencies for this Hook. 
```suggestion
	}, [ icon ] );
```